app/components/modal/FirstModal/FirstModal.module.css

```
@@ -0,0 +1,107 @@
+.firstmodalOverlay {
```

[キャメルケース](https://developer.mozilla.org/ja/docs/Glossary/Camel_case)に統一でお願いします！
```suggestion
.firstModalOverlay {
```